source/elements/oneDAL/source/glossary.rst

```diff
@@ -0,0 +1,117 @@
+.. _glossary:
+
+=========
+Glossary
+=========
+
+Machine learning terms
+======================
+
+.. glossary::
+    :sorted:
+
+    Attribute
+        A particular property or quality of a real object or an event.
+        Has a defined type and domain.
+
+    Classification
+        A supervised machine-learning problem to assign categorical :term:`labels<Label>`
+        to :term:`feature vectors<Feature vector>` describing input objects.
```

Comment:
* problem of assigning
* do you need "describing input objects" here? don't feature vectors always describe input objects?